cmd/serf/command/force_leave.go

```diff
@@ -33,14 +36,24 @@ func (c *ForceLeaveCommand) Run(args []string) int {
 		return 1
 	}
 
-	client, err := RPCClient(*rpcAddr, *rpcAuth)
+	RPCClient, err := RPCClient(*rpcAddr, *rpcAuth)
```

Comment:
Was there any particular reason for renaming this variable?

Reply:
[Yes, and no](x.co/yesandno) - it was for a different approach but I figured it couldn't _hurt_ to keep it renamed for clarity? Easy to undo if you think it's unnecessary.

Reply:
If you want to rename it, please don't have it start with an uppercase letter as that implies that it is exported, which is weird because it's a local variable. Probably just `rpcClient` then.